### ISSUE TITLE
outdated/conflicting assumed dir structure for SDK?

### ISSUE THREAD

**Issue by stuart-little** [flutter/flutter#62965]
I have described this on [Stack Overflow](https://stackoverflow.com/questions/63256612/android-sdk-tools-flutter-correct-directory-structure-and-environment-variabl), but it seems more like a Flutter issue.

It's not exactly a bug or deal breaker, but caused me confusion in trying to install.

## Steps to Reproduce

1. clone [this repo](https://github.com/flutter/flutter)
2. examine [the file](https://github.com/flutter/flutter/blob/master/dev/bots/download_android_sdk.sh)
```
dev/bots/download_android_sdk.sh
```
(henceforth to referred to as `reffile sh`). It expects the executable 

```
$ANDROID_SDK_ROOT/tools/bin/sdkmanager
```
3. examine [the file](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/android/android_sdk.dart)
```
packages/flutter_tools/lib/src/android/android_sdk.dart
```
(henceforth referred to as `reffile dart`). It expects e.g. the directory
```
$ANDROID_SDK_ROOT/build-tools
```
4. As detailed in [this SO post](https://stackoverflow.com/questions/60440509/android-command-line-tools-sdkmanager-always-shows-warning-could-not-create-se), the `sdkmanager` executable unzipped from the [officially-provided](https://developer.android.com/studio) Linux package `commandlinetools-linux-6609375_latest.zip` will *not* run unless the `tools` directory is unzipped into a higher-level `cmdline-tools` directory.

As per `reffile sh`, this means that after you've set up `sdkmanager` you want 
```
$ANDROID_SDK_ROOT/tools/bin/sdkmanager
=
<blah>/cmdline-tools/tools/bin/sdkmanager
```
or in other words
```
ANDROID_SDK_ROOT=<blah>/cmdline-tools
```
5. If I export that `ANDROID_SDK_ROOT` (*including* `cmdline-tools` in the path) and proceed to set up `SDK` (e.g. by running the likes of
```
$ANDROID_SDK_ROOT/tools/bin/sdkmanager platform-tools
```
)
then the resulting folders (`platform-tools`, `licenses`, etc.) populate at the same level as `$ANDROID_SDK_ROOT`, i.e. are siblings of it, all living inside 
```
<blah>
=
$ANDROID_SDK_ROOT/..
```
(note the double period).

6. This then conflicts with `reffile dart` above, which rather expects `platform-tools`, etc. to live *under* `$ANDROID_SDK_ROOT`.

---

Summary:

```
dev/bots/download_android_sdk.sh
```
and
```
packages/flutter_tools/lib/src/android/android_sdk.dart
```
appear to contradict each other on the expected `SDK` directory hierarchy.

## Logs

N/A: I do not believe log output is relevant here.

**Comment by TahaTesser** [comment]
I’m unable to reproduce the issue to pinpoint the cause. The issue needs further investigation.

**Comment by Hixie** [comment]
@stuart-little The files in dev/bots are specifically for our CI tools. Not intended to be used by developers. Though we might not even use that one any more.

**Comment by dnfield** [comment]
We need to remove that file :)  I'm pretty sure I had done that a ways back, and it broke something that was still referencing it, but it's not used anywhere and is outdated.

I'll repurpose this bug to do that, but if you're having some other meaningful problem with the Android SDK please feel free to file a new bug. Sorry for the confusion!

**Comment by stuart-little** [comment]
Good to know it *is* outdated! :)

When I first cloned the repo I had 0 experience with the `Android SDK` and poked around for any clue as to what `$ANDROID_SDK_ROOT` is supposed to be. I found that file and used it for guidance.

FYI: setting `ANDROID_SDK_ROOT=<blah>` in the setup above works just fine (`flutter doctor` finds everything, etc.). It's only this script that seems to conflict with that, but since you've confirmed it's to be removed anyway, I suppose closing this is the way to go.

Thanks!
